pkg/config/authorization/opa.go

```diff
@@ -81,3 +98,51 @@ func generatePolicyUID(policyName string, policyContent string, nonce int) strin
 	data := []byte(fmt.Sprint(nonce) + policyUIDHashSeparator + policyName + policyUIDHashSeparator + policyContent)
 	return fmt.Sprintf("%x", md5.Sum(data))
 }
+
+type ResponseOpaJson struct {
+	Result ResultJson `json:"result"`
+}
+
+type ResultJson struct {
+	Raw string `json:"raw"`
+}
+
+func downloadRegoDataFromUrl(url string) (string, error) {
+	if resp, err := http.Get(url); err != nil {
+		return "", fmt.Errorf("failed to fetch Rego config: %v", err)
+	} else {
+		defer resp.Body.Close()
+
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return "", fmt.Errorf("unable to read response body: %v", err)
+		}
+
+		if resp.StatusCode != http.StatusOK {
+			return "", fmt.Errorf("%s: %s", resp.Status, body)
+		}
+
+		result := string(body)
+		//json
+		if resp.Header["Content-Type"][0] == "application/json" {
+			var jsonResponse ResponseOpaJson
+			if err := json.Unmarshal(body, &jsonResponse); err != nil {
+				return "", fmt.Errorf("unable to unmarshal json response: %v", err)
+			}
+			result = jsonResponse.Result.Raw
+		}
+		return result, nil
+	}
+}
+
+func cleanUpRegoDocument(rego string) string {
+	r, _ := regexp.Compile("(\\s)*package.*")
+
+	lines := strings.Split(rego, "\n")
+	for i, line := range lines {
+		if r.MatchString(line) {
+			lines[i] = ""
+		}
+	}
+	return strings.Join(lines, "\n")
```

Comment:
Would this works perhaps?

```suggestion
	r, _ := regexp.Compile("(\\s)*package.*[;\\n\\s]+")
	return r.ReplaceAllString(s, "")
```

Reply:
Good catch, but I would remove the whitespace at the end of the regex, we want to catch the end of the line 
```regex
"(\\s)*package.*[;\\n]+"
```